### PR TITLE
DEV-671 Side Navigation redesign with light/dark mode

### PR DESCRIPTION
**Pull request by willmccormick24**
Updated side navigation with the redesign. Put IBC Transfer under Tools, updated external links, icons, and layout. Instead of turning all the SVGs into components I inverted` `tag based on theme. 

Light Mode (signed in):
Screenshot 2023-09-27 at 3 20 39 PM

Dark mode (signed out - tools opened):
Screenshot 2023-09-27 at 3 21 49 PM

### CODE REVIEW

**Comment by willmccormick24** [comment]
Oh one more thing: I was uncertain what the first external link was for on the Figma design. It was a globe icon marked as "language" but does Rebus app support multiple languages? I kept the old Medium icon/link in the meantime

**Review by nicolam1 — DISMISSED** [review]
LGTM

**Comment by nicolam1** [comment]
@willmccormick24 the PR looks good but I don't see the nav for EVM and Explorer 
Screenshot 2023-09-28 at 10 28 56 AM
Can we add those as well?

**Comment by willmccormick24** [comment]
> @willmccormick24 the PR looks good but I don't see the nav for EVM and Explorer Screenshot 2023-09-28 at 10 28 56 AM Can we add those as well?

@nicolam1 Sure no problem I can add those. I was going to add them later once Cosmos and EVM reskins were complete, but I'll update it now. Clicking on them should trigger external links to Explorer and EVM in the same window right?

**Comment by nicolam1** [comment]
@willmccormick24 you can use a feature flag to hide those menu items while we skin the other apps. Also please make the main URL they point to configurable so we can have different URLs for testnet and mainnet.

File: src/components/layouts/sidebar/index.tsx [hunk 1]
```diff
@@ -42,90 +43,98 @@ export const Sidebar: FunctionComponent<React.PropsWithChildren<unknown>> = obse
...
-									</div>
-								</section>
-								<section>
-									{mapKeyValues(LAYOUT.SIDEBAR, (_: string, sidebarItem: TSIDEBAR_ITEM) => sidebarItem)
-										.filter((sidebarItem: TSIDEBAR_ITEM) => {
-											if (sidebarItem.TYPE === 'assets') {
-												return featureFlagStore.featureFlags.assetsPage;
-											}
+		<SideBarStyled>
+			<React.Fragment>
```
Comment (Ericky14) [inline src/components/layouts/sidebar/index.tsx:47]: Remove React.Fragment if you're wrapping it

**Review by Ericky14 — CHANGES_REQUESTED** [review]
There's something weird going on in the sidebar, when you change routes the text blinks white

Oct-02-2023 11-17-05

**Comment by willmccormick24** [comment]
Added EVM/Explorer with a feature flag and configurable links. Also addressed Ericky's comments

Screenshot 2023-10-02 at 12 21 43 PM

**Review by Ericky14 — APPROVED** [review]

**Review by nicolam1 — APPROVED** [review]
LGTM

**Pull request closed without merging**